src/main/kotlin/hs/kr/equus/user/domain/auth/presentation/PassInfoController.kt

```diff
@@ -0,0 +1,27 @@
+package hs.kr.equus.user.domain.auth.presentation
+
+import hs.kr.equus.user.domain.auth.presentation.dto.request.PassPopupRequest
+import hs.kr.equus.user.domain.auth.presentation.dto.resopnse.QueryPassInfoResponse
+import hs.kr.equus.user.domain.auth.service.PassPopupService
+import hs.kr.equus.user.domain.auth.service.QueryPassInfoService
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+import javax.validation.Valid
+
+@RestController
+@RequestMapping("/user/verify")
+class PassInfoController(
+    private val passPopupService: PassPopupService,
+    private val queryPassInfoService: QueryPassInfoService
+) {
+
```

Comment:
🚫 **[ktlint]** <standard:no-empty-first-line-in-class-body> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class body should not start with blank line